appcircle_ios_objectivecswift_cache_push/1.0.2/component.yaml

```diff
@@ -0,0 +1,39 @@
+platform: iOS
+buildPlatform: ObjectiveCSwift
+displayName: "Cache Push"
+description: "Uploads user selected files and folders to Appcircle cache."
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#cache-push
+requiredComponents: "appcircle_cocoapods"
+precedingComponents: "appcircle_ios_test,appcircle_ios_build_simulator,appcircle_ios_build_sign"
```

Comment:
> [So, we should not include any preceding or required component for "Cache Push" based on the default values in "Included Paths".](https://github.com/appcircleio/appcircle-workflow-components/pull/352#issuecomment-1973043160)

@boztopuz  Onur gave the feedback above. Could you please make changes accordingly?